packages/database/src/migrations/20210615035418-DropVillagesFromLESMIS-modifies-data.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getHierarchyId = async (db, name) => {
+  const { rows: data } = await db.runSql(`
+    SELECT id FROM entity_hierarchy
+    WHERE name = '${name}'
+  `);
+  return data[0]?.id;
+};
+
+exports.up = async function (db) {
+  const hierarchy = await getHierarchyId(db, 'laos_schools');
+  await db.runSql(`
+  UPDATE entity_hierarchy
+  SET canonical_types = array_remove(canonical_types, 'village')
+  WHERE name = 'laos_schools'
+  `);
```

Comment:
I tried just removing the `canonical_type` but the entity server was still pulling villages from the other tables.

I'm not 100% sure about all the possible side effects deleting the `entity_relations` and `ancestor_descendant_relations`, but it works in testing